go.mod

```diff
@@ -432,3 +432,5 @@ require (
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	golang.org/x/exp v0.0.0-20240904232852-e7e105dedf7e
 )
+
+replace go.viam.com/api => ./api
```

Comment:
For https://github.com/viamrobotics/api/pull/634. Will remove once `api` PR is merged.